eZ/Publish/Core/Persistence/Cache/UserHandler.php

```diff
@@ -215,6 +215,23 @@ public function update(User $user)
         return $user;
     }
 
+    public function updatePassword(User $user)
```

Comment:
Interface that this class implements is missing return type for this API. It should be added even if other methods of this interface do not have them?

Reply:
Anyway, return type added.

Reply:
> Interface that this class implements is missing return type for this API. It should be added even if other methods of this interface do not have them?

It depends. If we have control over the code, then definitely. In other cases it needs to be assessed (for instance if 3rd party type-hints in PHPDoc e.g. `@return bool` then it can be safely applied as well).